apps/bynder-content-workflow/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "gather-content-app",
```

Comment:
Is this the intended name of the app? Just want to make sure it's not supposed to be "bynder-content-workflow"